runners/reference/job-server/build.gradle

```diff
@@ -34,7 +35,7 @@ description = "Apache Beam :: Runners :: Reference :: Job Server"
 dependencies {
   compile project(path: ":beam-runners-direct-java", configuration: "shadow")
   compile project(path: ":beam-runners-java-fn-execution", configuration: "shadow")
-  compile library.java.slf4j_simple
+  shadowTestRuntimeClasspath library.java.slf4j_simple
```

Comment:
I believe the reference job server is also meant to be a standalone application and we want the logging backend to be included so that logging is output.

Reply:
done